packages/packages/v1-adapters/src/dispatchers/dispatchers.ts

```diff
@@ -37,8 +37,14 @@ export function openRoute( route: string ): Promise<void> {
 	}
 }
 
-export function isRouteActive( route: string ): boolean {
+export function isRouteActive( route: string ) {
 	const extendedWindow = window as unknown as ExtendedWindow;
 
 	return !! extendedWindow.$e?.routes?.isPartOf( route );
 }
+
+export function getCurrentEditMode() {
+	const extendedWindow = window as unknown as ExtendedWindow;
+
+	return extendedWindow.elementor.channels.dataEditMode.request( 'activeMode' );
```

Comment:
```suggestion
	return extendedWindow?.elementor?.channels?.dataEditMode?.request( 'activeMode' );
```

to support using the code without the editor v1